### DISCUSSION TITLE
One analog ROl wrong values

### DISCUSSION THREAD

**Discussion by LarsSche** [jomjol/AI-on-the-edge-device#1562] [Q&A] [answered]
solved the problem ...

"Counter-Clockwise Rotation" was active.
Disabled and the values are ok.

Hi

my problem is that one analog ROl gives wrong values.
The other 3 are ok with the values.
What can I do?
Attached a picture of current values.
The ROl far left.
 
![Bildschirm­foto 2022-12-12 um 13 42 19](https://user-images.githubusercontent.com/71009342/207050857-a36f36bd-14ef-4699-8312-6c0092d04ade.png)

**Comment by jomjol** [comment, marked as answer]
Well, the numbers are not too bad. I assume you mean the 4.9 to be wrong. But +/- 0.5 is accetable.

But you can improve, if you make the ROI better. The outer green circle should fit to the black circle of the analog scala. That might make it even better.

**Reply by LarsSche** [reply]
Ah ok... Will do that.
But the problem was that the ROl was configured with "Counter-Clockwise Rotation"
Disabled the option and the values are ok.
Will try it with the green cycle anyway
